packages/patient-registry/app/models/queries.ts

```diff
@@ -2,75 +2,124 @@ import config from "@amrs-integrations/core";
 
 export async function getFacilityMfl(param: string) {
   let CM = config.ConnectionManager.getInstance();
-  let amrsCON = await CM.getConnectionAmrs();
+  let amrsCON: any;
```

Comment:
What's the role of this variable? You might want to instantiate the connection here and do a releaseConnection(amrsCon) after obtaining the results